### PR TITLE
feat: initial event creation page

### PR DESCRIPTION
**Pull request by AlbertoCentonze**
# What I did

Implemented a state-less event creation screen to connect to the database.

# How I did it

- Created a new screen, connected to the `Stack.Navigator` for logged in users and created some basic styling.

# How to verify it

1. Login
2. Go to settings screen
3. Click on create event
4. Try the page out as in the demo

# Demo video
https://github.com/uniconnect-epfl/uniconnect/assets/11707683/f1d7ce0c-d1ba-413f-9874-60be0921d768

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [x] work correctly on Android
- [x] are fully tested

### CODE REVIEW

**Review by GustaveCharles — CHANGES_REQUESTED** [review]

File: navigation/Main/MainStackNavigator.tsx [hunk 1]
```diff
@@ -88,7 +89,7 @@ const MainStackNavigator: React.FC = () => {
     >
       <Stack.Navigator
         initialRouteName={
-          user ? (fillOutProfile ? "UpdateProfile" : "HomeTabs") : "Onboarding"
+          user ? (fillOutProfile ? "EventCreation" : "HomeTabs") : "Onboarding"
```
[the remark this answers is not in the document]
Comment (AlbertoCentonze) [reply navigation/Main/MainStackNavigator.tsx:92]: Indeed this was left for testing, my bad!

**Review by GaelCondeLosada — APPROVED** [review]
LGTM, everything works on my android device ! Two questions but that's really minor stuff I think you can merge

File: screens/EventCreation/EventCreationScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  Alert,
+} from "react-native"
+import { styles } from "./styles"
+import { useNavigation } from "@react-navigation/native"
+import { Ionicons } from "@expo/vector-icons"
+import { globalStyles } from "../../assets/global/globalStyles"
+import { peach, white } from "../../assets/colors/colors"
+
+const EventCreationScreen = () => {
+  const navigation = useNavigation()
```
[the remark this answers is not in the document]
Comment (AlbertoCentonze) [reply screens/EventCreation/EventCreationScreen.tsx:17]: I don't think so, that's the whole point of using hooks (avoid props drilling). I'll @Aidasvenc confirm.

[the remark this answers is not in the document]
Comment (GaelCondeLosada) [reply screens/EventCreation/EventCreationScreen.tsx:17]: Ok I learnt something today.

**Review by GustaveCharles — APPROVED** [review]
LGTM. Thanks a lot for this implementation!

**Pull request merged**